### DISCUSSION TITLE
Alert Placement

### DISCUSSION THREAD

**Discussion by eleifer-twilio** [twilio-labs/paste#1042] [General] [answered]
This is how Phone numbers alerts look today. Does this follow Paste guidelines or should the alert placements be above the title? @serifluous 

![Screen Shot 2020-12-14 at 1 17 44 PM](https://user-images.githubusercontent.com/73968607/102137725-2240c100-3e10-11eb-8b29-e16d58376326.png)

**Comment by TheSisb** [comment, marked as answer]
Hi @eleifer-twilio,

We have a `Do & Don't` section on our docs which may help answer your question: https://paste.twilio.design/components/alert#when-to-use-an-alert

![image](https://user-images.githubusercontent.com/1592327/102240064-778ada00-3ec5-11eb-8680-a036fdb50c81.png)
